solver/pb/caps.go

```diff
@@ -212,6 +213,12 @@ func init() {
 		Status:  apicaps.CapStatusExperimental,
 	})
 
+	Caps.Init(apicaps.Cap{
+		ID:      CapMountSSH,
```

Comment:
nit: Should be `CapExecMountSSH`. The secret one is incorrectly named as well.

Reply:
is this only for exec?

Reply:
I mean git source op

Reply:
Yes, we'll add another cap when git support is implemented.